src/EditorFeatures/Core/Implementation/InlineRename/InlineRenameSession.OpenTextBufferManager.cs

```diff
@@ -308,10 +309,27 @@ internal void Disconnect(bool documentIsClosed, bool rollbackTemporaryEdits)
                 }
             }
 
+            private Task<LinkedFileMergeSessionResult> ComputeMergeResultAsync(Solution newSolution, CancellationToken cancellationToken)
+            {
+                var preMergeSolution = _session._baseSolution;
+
+                var diffMergingSession = new LinkedFileDiffMergingSession(preMergeSolution, newSolution, newSolution.GetChanges(preMergeSolution), logSessionInfo: true);
+                return diffMergingSession.MergeDiffsAsync(mergeConflictHandler: null, cancellationToken: cancellationToken);
+            }
+
+            [Obsolete("Update code to call ComputeMergeResultAsync prior to calling the other overload of ApplyConflictResolutionEdits.")]
```

Comment:
Concerned something may be IVT referencing this and didn't want to risk a break so late in the release.